src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -545,6 +544,7 @@ protected void VisitLvalue(BoundExpression node)
                     }
 
                 case BoundKind.TupleLiteral:
```

Comment:
>TupleLiteral [](start = 31, length = 12)

can we still get an unconverted tuple literal here?

Reply:
Yes, in error scenarios.

---
In reply to: [301813107](https://github.com/dotnet/roslyn/pull/37052#discussion_r301813107) [](ancestors = 301813107)